--- ./libjpeg/jmorecfg.h	Thu Oct  7 16:38:56 2004
+++ ../../../horizon.orig/lib/libjpeg/jmorecfg.h	Wed Aug 18 12:18:08 2004
@@ -158,7 +158,7 @@
 /* INT32 must hold at least signed 32-bit values. */
 
 #ifndef XMD_H			/* X11/xmd.h correctly defines INT32 */
-typedef long INT32;
+// typedef long INT32;
 #endif
 
 /* Datatype used for image dimensions.  The JPEG standard only supports
@@ -210,9 +210,9 @@
  */
 
 #ifdef NEED_FAR_POINTERS
-#define FAR  far
+// #define FAR  far
 #else
-#define FAR
+// #define FAR
 #endif
 
 
@@ -224,7 +224,7 @@
  */
 
 #ifndef HAVE_BOOLEAN
-typedef int boolean;
+// typedef int boolean;
 #endif
 #ifndef FALSE			/* in case these macros already exist */
 #define FALSE	0		/* values of boolean */
