--- ./zlib/zconf.h.orig	Thu Oct  7 16:05:52 2004
+++ ./zlib/zconf.h	Thu Oct  7 16:06:11 2004
@@ -275,7 +275,7 @@
    typedef Byte       *voidp;
 #endif
 
-#if 1           /* HAVE_UNISTD_H -- this line is updated by ./configure */
+#if 0           /* HAVE_UNISTD_H -- this line is updated by ./configure */
 #  include <sys/types.h> /* for off_t */
 #  include <unistd.h>    /* for SEEK_* and off_t */
 #  ifdef VMS
